### DISCUSSION TITLE
v0.9.0-test2

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#1643] [General]
Pre-release version 2 of plugdata v0.9.0

Headlines:
- Huge performance improvements because of GPU accelerated rendering!
- Design improvements: less outlines, smaller toolbar, reorganised statusbar, new about panel.
- Ships with Gem built in. Gem objects need the "Gem/" prefix, unless you add Gem to the list of libraries to load.
- Improved presentation mode
- Added macOS legacy version, recommended when running macOS <10.15 or on Macs from before 2017
- New connection activity display
- Connection arrows drawn between segments when connection is segmented
- Windows build now uses system rounded window border and dropshadow. (Straight for w10)
- Connections can be set behind / in front for both edit & locked mode.
- Can now search for send/receive symbols in patch search panel
- Rewritten tabbar and split view for better stability and maintainability
- Improved patch loading time in DAW.
- Patches load a lot faster now
- Latest ELSE master branch changes, including updated MERDA modules
- Latest pure-data master branch changes
- Improved compatibility with [drawcurve], [fillcurve] and [plot] drawing
- Redesigned GraphOnParent area resizer
- Plugin version now displays the current latency in the statusbar if it has been changed from the default
- Improvements to pd-vanilla compatibility, patches should look even more similar to their vanilla counterparts than before
- Improved Heavy toolchain download speed
- Fixed [keyboard] bugs
- Many bugfixes!
- A lot I'm forgetting here, since this has been 6 months in the making

Breaking changes:
- [cyclone/scope~] and [else/oscope~] have been replaced with [else/scope~]

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/v0.9.0-test2'>v0.9.0-test2</a>.</em>

**Comment by AlexMitchellMus** [comment]
A few additional changes:

* symbol iolets are now patchable even after a symbol has been set for the iolet
* all objects now correctly hide their symbol iolets
* Keyboard object now only shows octave numbers in edit mode
